### PR TITLE
API Set default value of framework RememberLoginHash logout_across_devices to false

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue: https://github.com/silverstripe/silverstripe-session-manager/issues/65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,24 @@ It is also compatible with the [Silverstripe MFA module suite](https://github.co
 
 ## Configuration
 
+### Logout across devices
+
+This module will update the `SilverStripe\Security\RememberLoginHash.logout_across_devices` config setting, setting it to `false` by default.
```
Comment (maxime-rainville) [inline README.md:58]:
```suggestion
This module updates the `SilverStripe\Security\RememberLoginHash.logout_across_devices` configuration flag to default to `false`.
```

Comment (emteknetnz) [reply README.md]: Have updated text

File: README.md [hunk 2]
```diff
@@ -53,6 +53,24 @@ It is also compatible with the [Silverstripe MFA module suite](https://github.co
 
 ## Configuration
 
+### Logout across devices
+
+This module updates the `SilverStripe\Security\RememberLoginHash.logout_across_devices` configuration flag to default to `false`.
+
+This is done because session manager is designed to provide a good way to revoke sessions for devices that you do not want to be logged in.  Without session manager, the only way to revoke a different device was the slightly odd method of logging out of your current device, which in turn deleted the database record created when a user checked the "Remember me" checkbox when logging in to the admin section. This is considered to be an odd UX experience if you are legitimately logged in on multiple devices.
+
+To restore the old behaviour with session manager installed, add the following YML config to your project:
```
Comment (maxime-rainville) [inline README.md]: I don't think we need to go into this level of detail here.

There's a already a [bit about this config flag in the regular doc](https://docs.silverstripe.org/en/4/developer_guides/security/member/#saved-user-logins). If this does not provide enough info, then it probably make more sense to update it there.

I would be tempted to keep it short and sweet. Maybe something like:
> Session-manager provides an explicit way to terminate individual sessions and their attached "remember me" tokens. So this module sets `SilverStripe\Security\RememberLoginHash.logout_across_devices` to `false`.
>
> To restore the old behaviour with session manager installed, add the following YML config to your project:
> ...
> Read [Saved User Logins](https://docs.silverstripe.org/en/4/developer_guides/security/member/#saved-user-logins) to learn how to configure the "remember me" feature for your users.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Looks good ... tested the new behaviour with and without the code snippet in the doc.

My instinct would be to keep the doc a bit more simple, but I don't have strong feelings about it.

File: README.md (hunk 2 above)
Comment (emteknetnz) [reply README.md]: Updated to your text

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**